routers/biocommons_groups.py

```diff
@@ -90,6 +90,9 @@ def approve_group_access(
     approving_user: Annotated[SessionUser, Depends(get_session_user)],
     db_session: Annotated[Session, Depends(get_db_session)],
     auth0_client: Annotated[Auth0Client, Depends(get_auth0_client)],
+    background_tasks: BackgroundTasks = None,
```

Comment:
This should just be `background_tasks: BackgroundTasks`, I don't think we should have a default of `None`